tests/models/test_taskinstance.py

```diff
@@ -3421,14 +3422,27 @@ def on_execute_callable(context):
             ti.refresh_from_db()
             assert ti.state == State.SUCCESS
 
-    def test_finished_callbacks_handle_and_log_exception(self, caplog):
+    def test_finished_callbacks_callables_handle_and_log_exception(self, caplog):
+        def on_finish_callable(context):
+            pass
+
+        for callback_input in [[on_finish_callable, on_finish_callable], on_finish_callable]:
```

Comment:
Let’s use `pytest.mark.parametrize` instead of a for loop

Reply:
let me check the api

Reply:
I'll check parametrize later.
But add enumerate won't fix the test.

Instead, need to move stdout recorder out of the loop.